graphql/resolvers/_RootQuerys/me.js

```diff
@@ -30,7 +28,10 @@ module.exports = async (_, args, { user }) => {
   return {
     ...user,
     name,
+    address: await pgdb.public.addresses.findOne({ id: user.addressId }),
```

Comment:
This calls for a separate User/-address resolver. The advantage is that this DB roundtrip can be saved if the field is not requested. Checkout graphql/resolvers/Comment.js

Reply:
Done. LGTY?